src/components/App/App.js

```diff
@@ -98,72 +93,42 @@ export default function App() {
         song.genre === values.genre
     )
 
-    const shuffledWarmupSongs = allSongsForWarmup.sort(
-      () => 0.5 - Math.random()
-    )
+    //  const allSongsForWarmup = filterAllSongsForSection(
+    //   data,
+    //   newWorkout.warmup.cadence,
+    //   values.genre
```

Comment:
In your `filterAllSongsForSection` function you get the genre from your values in line 13.
```suggestion
    //   values
```